readthedocs/builds/managers.py

```diff
@@ -179,3 +179,37 @@ class InternalBuildManager(SettingsOverrideObject):
 
 class ExternalBuildManager(SettingsOverrideObject):
     _default_class = ExternalBuildManagerBase
+
+
+class VersionAutomationRuleManager(PolymorphicManager):
+
+    def append_rule(
+        self, *, project, description, match_arg, version_type,
+        action, action_arg=None,
+    ):
+        """
+        Append an automation rule to `project`.
+
+        The rule is created with a priority higher than the last rule
```

Comment:
Reading the code, it seems it's created with a "lower priority" since it will be executed as the last one. The highest priority is the `0`.

Maybe you can re-write the docstring to avoid this problem.